src/main/java/pl/javastart/task/phone/Contract.java

```diff
@@ -0,0 +1,67 @@
+package pl.javastart.task.phone;
+
+public abstract class Contract {
+    private double accountValue;
```

Comment:
Ze względu na to, że w zależności od typu umowy, to pole oznacza różne rzeczy, to nie definiowałbym go tutaj w klasie nadrzędnej